libraries/AP_RPM/RPM_Generator.h

```diff
@@ -0,0 +1,37 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "AP_RPM.h"
+#include "RPM_Backend.h"
+#include <AP_Generator/AP_Generator.h>
+
+#if HAL_GENERATOR_ENABLED
+
+class AP_RPM_Generator : public AP_RPM_Backend
+{
+public:
+    // constructor
+    AP_RPM_Generator(AP_RPM &ranger, uint8_t instance, AP_RPM::RPM_State &_state);
+
+    // update state
+    void update(void) override;
+
+private:
+    uint8_t instance;
```

Comment:
This doesn't do anything? The same seems true for the ESC backend..... Removing it would allow `using AP_RPM_Backend::AP_RPM_Backend`

Reply:
thanks, fixed